examples/sampling-adaptive-covariance-rao-blackwell.ipynb

```diff
@@ -0,0 +1,150 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Inference: Adaptive Covariance Rao-Blackwell MCMC\n",
+    "\n",
+    "This example shows you how to perform Bayesian inference on a time series, using a variant of [Adaptive Covariance MCMC](http://pints.readthedocs.io/en/latest/mcmc_samplers/adaptive_covariance_mcmc_remi.html) detailed in Algorithm 3 of [1].\n",
+    "\n",
+    "[1] A tutorial on adaptive MCMC\n",
```

Comment:
Make this a link to https://doi.org/10.1007/s11222-008-9110-y ?

Reply:
Done